tests/v1/kv_connector/run_accuracy_test.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -xe
+
+# Model to run.
+MODEL_NAME=Qwen/Qwen3-0.6B
+
+# Trap the SIGINT signal (triggered by Ctrl+C)
+trap 'kill $(jobs -pr)' SIGINT SIGTERM EXIT
+
+# Cleanup function
+cleanup() {
+    echo "Caught Ctrl+C, cleaning up..."
+    # Cleanup commands
+    pgrep python | xargs kill -9
+    pkill -f python
+    echo "Cleanup complete. Exiting."
+    exit 0
+}
```

Comment:
It looks like the cleanup function is never used -- should we remove it? It looks like `trap 'kill $(jobs -pr)' SIGINT SIGTERM EXIT` will do the job (but we could think about adding a `-9`)